cmd/coder/urls.go

```diff
@@ -193,7 +223,7 @@ func urlList(envName string) []DevURL {
 	}
 
 	if len(devURLs) == 0 {
-		fmt.Printf("no dev urls were found for environment: %s\n", envName)
+		flog.Error("no dev urls were found for environment: %s", envName)
```

Comment:
Unrelated to this review: It's a bit confusing to the user we *always* log an error in red to the console if there are no devurls here. This function is used to get the list of existing devurls, which can legitimately be empty. I'll look at moving the error handling up to where we know whether to expect an empty list or not, if that makes sense.